services/QuillLMS/app/models/concept_feedback.rb

```diff
@@ -21,17 +21,26 @@ class ConceptFeedback < ApplicationRecord
     TYPE_CONNECT = 'connect',
     TYPE_GRAMMAR = 'grammar'
   ]
+
+  ALL_CONCEPT_FEEDBACKS_KEY = 'all_concept_feedbacks'
+
   validates :data, presence: true
   validates :uid, presence: true, uniqueness: { scope: :activity_type }
   validates :activity_type, presence: true, inclusion: {in: TYPES}
   validate :data_must_be_hash
 
+  after_commit :clear_concept_feedbacks_cache
+
   def as_json(options=nil)
     data
   end
 
```

Comment:
The cache key seems to be used in several places, so I would extract it into its own method:

```
def foo = "#{ALL_CONCEPT_FEEDBACKS_KEY}_#{activity_type}"
```